src/pages/[platform]/build-a-backend/more-features/in-app-messaging/in-app-messaging-migration-guide/index.mdx

```diff
@@ -39,138 +39,545 @@ export function getStaticProps(context) {
+
+    await clearMessages();
+    ```
+
+  </Block>
+  <Block name="V5">
+    ```ts
+    import { Notifications } from 'aws-amplify';
+    const { InAppMessaging } = Notifications;
+
+    await InAppMessaging.clearMessages();
+    ```
+
+  </Block>
+</BlockSwitcher>
+
+## InAppMessaging.identifyUser
+
+The `identifyUser` API in v6 has not changed in behavior; however, it is now exported from the `aws-amplify/in-app-messaging` path. Additionally, the input parameters have been updated as follows:
+
+- Instead of two position positional parameters (corresponding to `userId` and `userInfo`), there are now three named parameters: `userId`, `userProfile` and `options` (i.e. a single input object containing these properties).
+- The `attributes` property previously found under `userInfo` has been renamed to `customProperties` and can now be found under `userProfile`.
+- A new `userAttributes` property can be found under `options`.
+- New convenience properties `name`, `email` and `plan` can be found under `userProfile`. These properties are automatically merged into `customProperties`.
+
+<Accordion title='Input' headingLevel='3'>
+  <Columns columns={2}>
+    <div>
+      **V5**
+
+      ```
+      userId: string;
+      userInfo: {
+        attributes?: Record<string, string[]>;
+        demographic?: {
+          appVersion?: string;
+          locale?: string;
+          make?: string;
+          model?: string;
+          modelVersion?: string;
+          platform?: string;
+          platformVersion?: string;
+          timezone?: string;
+        };
+        location?: {
+          city?: string;
+          country?: string;
+          latitude?: number;
+          longitude?: number;
+          postalCode?: string;
+          region?: string;
+        };
+        metrics?: Record<string, number>;
+      }
+      ```
+
+    </div>
+    <div>
+      **V6**
+
+      ```
+      input: {
+        userId: string;
+        userProfile: {
+          customProperties?: Record<string, string[]>;
+          demographic?: {
+            appVersion?: string;
+            locale?: string;
+            make?: string;
+            model?: string;
+            modelVersion?: string;
+            platform?: string;
+            platformVersion?: string;
+            timezone?: string;
+          };
+          email?: string;
+          location?: {
+            city?: string;
+            country?: string;
+            latitude?: number;
+            longitude?: number;
+            postalCode?: string;
+            region?: string;
+          };
+          metrics?: Record<string, number>;
+          name?: string;
+          plan?: string;
+        };
+        options?: { userAttributes?: Record<string, string[]>; };
+      }
+      ```
+
+    </div>
+  </Columns>
+</Accordion>
+
+<BlockSwitcher>
+  <Block name="V6">
+    ```ts
+    import { identifyUser } from 'aws-amplify/in-app-messaging';
+
+    const identifyUserInput = {
```

Comment:
These code blocks might be easier to read if we put the values into their properties and removed the comments.